.circleci/config.yml

```diff
@@ -106,6 +106,7 @@ jobs:
       - *attach_workspace
       - run: make api.install
       - run: make api.build
+      - run: make -C api newrelic.ini
```

Comment:
Do we mean something like `make api.setup` above `api.install`? And inside that, it would write out the ini file.

Reply:
Could write out the ini file in `make api.install` but I added `api.setup` as it seems more of a setup thing.

Reply:
> but I don't want it to generate the newrelic.ini on developer machines.

Fair.

Yeah we probably would revert it and then either add it back it for further testing, or work out how to enable it for certain envs if we want to keep it in.